onnxruntime/test/testdata/reduced_ops_via_config.readme.txt

```diff
@@ -0,0 +1,10 @@
+reduced_ops_via_config.onnx_model_with_included_ops.config contains the ops from
+reduced_ops_via_config.onnx_model_with_included_ops, which is intentionally named 
+to not have a '.onnx' extension so that the reduced build configuration generation script ignores it. 
+The model involves some ops that are not present in any other testdata models (specifically ReduceProd), 
+so failure to include the information in the configuration file in a minimal build should result in
+ that model being unable to be loaded.
```

Comment:
>  [](start = 0, length = 1)

nit: leading space
are we relying on other testdata models to not have ReduceProd? if so, should we enforce this somehow?

Reply:
I don't know that there's much value in this test anymore. It was previously testing building with both a config and a models dir, but we only allow building with a config now. Probably better to delete instead of adding some enforcement mechanism.